tests/geometry/test_tools.py

```diff
@@ -846,3 +847,49 @@ def get_shared_fragments(group_1, group_2):
                 if sol.is_same(sol_2):
                     count += 1
         return count
+
+
+class TestForceWireToSpline:
+    filename = "vvts_face_polygon_test.json"
+
+    @classmethod
+    def setup_class(cls):
+        path = get_bluemira_path("geometry/test_data", subfolder="tests")
+        filename = os.path.join(path, cls.filename)
+        wires = []
+        with open(filename, "r") as file:
+            data = json.load(file)
+
+        for wire in data["BluemiraFace"]["boundary"]:
+            wires.append(deserialize_shape(wire))
+        cls.wires = wires
+
+    @mock.patch("bluemira.geometry.tools.bluemira_warn")
+    def test_force_spline(self, bm_warn):
```

Comment:
Its overkill here but you can also use the pytest caplog fixture for checking logging output instead of using the mock.  
`def test_force_spline(self, caplog):`